app/src/test/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModelTest.kt

```diff
@@ -414,6 +416,36 @@ class GroupConversationDetailsViewModelTest {
         assertEquals(expected, viewModel.conversationSheetContent)
     }
 
+    @Test
```

Comment:
Maybe we can `verify` that the mocked usecase is called, so we know later if something is not called, we can catch it 🦋